app/src/main/java/org/p2p/wallet/renbtc/interactor/BurnBtcInteractor.kt

```diff
@@ -48,25 +45,33 @@ class BurnBtcInteractor(
             Account(signerSecretKey)
         )
 
-        val burnState = getBurnState(burnDetails, amount.toString())
-        val hash = try {
-            release()
-        } catch (e: RpcException) {
-            // TODO: Handle this error [invalid burn info: cannot get burn info: decoding solana burn log: expected data len=97, got=0]
-            // TODO: It crashes even if transaction is valid
-            if (e.message?.startsWith("invalid burn info") == true) {
-                return@withContext burnDetails.confirmedSignature
-            } else {
-                throw e
-            }
-        }
+        // TODO: We are not using state and hash.
+        // TODO: WORKAROUND. it's crashing anyway. Temporary commenting it.
+//        val burnState = try {
+//            getBurnState(burnDetails, amount.toString())
+//        } catch (e: Throwable) {
+//            // TODO: We are not using state, therefore ignoring errors for now
+//            Timber.e(e, "Error getting state")
+//        }
+
+//        val hash = try {
+//            release()
+//        } catch (e: RpcException) {
+//            // TODO: Handle this error [invalid burn info: cannot get burn info: decoding solana burn log: expected data len=97, got=0]
+//            // TODO: It crashes even if transaction is valid
+//            if (e.message?.startsWith("invalid burn info") == true) {
+//                return@withContext burnDetails.confirmedSignature
+//            } else {
+//                throw e
+//            }
+//        }
```

Comment:
don't worry, transaction is successful anyway
https://explorer.solana.com/tx/4L7L4FDoNRt62zB23DZeVTXb5S5c8UpsbJvmJ1TkPMqN8hRu8TAkhjq8ApGZn82SbRrMgDZniFwyoKN6mC4fhBV6